bricks/test_optimizer/brick.yaml

```diff
@@ -1,6 +1,6 @@
 name: test_optimizer
 description: A brick that generates a single entrypoint for Dart tests.
-version: 0.1.0+1
+version: 0.1.0+2
```

Comment:
We dont publish this brick, maybe we should even remove the version.

/cc @renancaraujo @felangel

Reply:
I'll set it back to 0.1.0+1 since removing violates the brick schema. Wdyt?